skin/src/vue/c-content.vue

```diff
@@ -165,44 +165,46 @@ onUpdated(() => {
     <div class="doc-header">
       <div class="doc-header-inner">
         <h1 class="doc-title">{{ title }}</h1>
-        <div class="doc-info">
-          <span class="doc-info-user-info">
+        <div class="info-wrapper">
+          
+          <span class="doc-author">
             <x-avatar class="avatar" :image="avImg" size="2rem"></x-avatar>
             User Name edited on 12/12/2024 at 12:00
           </span>
-          <!-- TODO: add a way to inject those by extension
-               and provide one for the number of attachments.
-              It must be reactive whenever the attachment store is updated -->
-          <div class="doc-info-actions">
-            <suspense>
-              <info-actions></info-actions>
-            </suspense>
-            <div class="doc-page-actions">
-              <router-link
-                :to="
-                  currentPageRevision
-                    ? ''
-                    : {
-                        name: 'edit',
-                        params: { page: currentPageName },
-                      }
-                "
-              >
-                <x-btn
-                  size="small"
-                  :disabled="currentPageRevision !== undefined"
+            <!-- TODO: add a way to inject those by extension
+                 and provide one for the number of attachments.
+                It must be reactive whenever the attachment store is updated -->
+            <div class="doc-info-actions">
+              <suspense>
+                <info-actions></info-actions>
+              </suspense>
+          
+              <div class="doc-page-actions">
+                <router-link
+                  :to="
+                    currentPageRevision
+                      ? ''
+                      : {
+                          name: 'edit',
+                          params: { page: currentPageName },
+                        }
+                  "
                 >
-                  <c-icon name="pencil" :size="Size.Small"></c-icon>
-                  Edit
-                </x-btn>
-              </router-link>
-              <page-actions
-                :current-page="currentPage"
-                :current-page-name="currentPageName"
-                :disabled="currentPageRevision !== undefined"
-              ></page-actions>
+                  <x-btn
+                    size="small"
+                    :disabled="currentPageRevision !== undefined"
+                  >
+                    <c-icon name="pencil" :size="Size.Small"></c-icon>
+                    Edit
+                  </x-btn>
+                </router-link>
+                <page-actions
+                  :current-page="currentPage"
+                  :current-page-name="currentPageName"
+                  :disabled="currentPageRevision !== undefined"
+                ></page-actions>
             </div>
-          </div>
+        </div>
         </div>
```

Comment:
Seems like the internal div and its content are missing an indent.
```suggestion
          </div>
        </div>
```